rs/nns/governance/src/lib.rs

```diff
@@ -140,7 +140,7 @@ use std::{
     time::{Duration, SystemTime},
 };
 
-#[cfg(test)]
+#[cfg(any(test, feature = "canbench-rs"))]
```

Comment:
hmm, might be a bit late but is it a good idea to add another rust compilation flag?

Reply:
not sure I understand your objection.  However, this is supposed to be in the next PR in the series...